src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -875,24 +601,23 @@ public static List<VariantContext> getVariantContextsFromGivenAlleles(final int
      * @param haplotypes list of active haplotypes at the current location
      * @param includeSpanningEvents If true, will also return events that span loc
      */
-    public static List<VariantContext> getVariantContextsFromActiveHaplotypes(final int loc,
-                                                                                 final List<Haplotype> haplotypes,
-                                                                                 final boolean includeSpanningEvents) {
-        final List<VariantContext> results = new ArrayList<>();
-        final Set<LocationAndAlleles> uniqueLocationsAndAlleles = new HashSet<>();
-
-        haplotypes.stream()
-                .flatMap(h -> Utils.stream(h.getEventMap().getOverlappingEvents(loc)))
-                .filter(Objects::nonNull)
-                .filter(v -> (includeSpanningEvents || v.getStart() == loc))
-                .forEach(v -> {
-                    final LocationAndAlleles locationAndAlleles = new LocationAndAlleles(v.getStart(), v.getAlleles());
-                    if (! uniqueLocationsAndAlleles.contains(locationAndAlleles)) {
-                        uniqueLocationsAndAlleles.add(locationAndAlleles);
-                        results.add(v);
-                    }
-                });
-        return results;
+    public static List<VariantContext> getEventsFromActiveHaplotypes(final int loc, final List<Haplotype> haplotypes, final boolean includeSpanningEvents) {
```

Comment:
this seems like a misnomer given that this is the exit point for returning VCs from the events...

Reply:
fixed